ithaka/aiw-ui
# Air 1108

To test:
- Navigate to Collections, then select Groups from the Browse page nav (should show Institutional for Sahara)
- Shouldn't be able to browse to the "Search" Groups page
